src/app/templates/app/license_requests.html

```diff
@@ -70,14 +73,29 @@
     });
     $('body').on('click', 'button[id^="archive_button"]', function (en) {
       en.stopImmediatePropagation();
-      $("#modal-header").addClass("yellow-modal");
-	    $(".modal-footer").html('<button class="btn btn-success" id="confirm_archive"><span class="glyphicon glyphicon-off"></span> Confirm</button>');
-      $("#modal-body").html('Are you sure you want to archive this license?');
-      $("#myModal").modal({
-        backdrop: 'static',
-        keyboard: true,
-        show: true,
-      })
+      var githubLogin =  "{{ github_login }}" ;
+      if ((githubLogin) != "None"){
+        if (" {{ authorized }} ") {
+          $("#modal-header").addClass("yellow-modal");
+          $("#modal-body").html('Are you sure you want to archive this license?');
+          $("#myModal").modal({
+                backdrop: 'static',
+                keyboard: true,
+                show: true,
+          })
+        }
+        else{
+              $("#modal-header").addClass("yellow-modal");
+	            $(".modal-footer").html('<button class="btn btn-success" id="confirm_archive"><span class="glyphicon glyphicon-off"></span> Confirm</button>');
+              $("#modal-body").html('You are not permitted to archive this request.');
+              $("#myModal").modal({
+                backdrop: 'static',
+                keyboard: true,
+                show: true,
+          })
+       }
+      }
```

Comment:
Can this check be moved in backend?
On Ui we don't show the button for user who are not logged in and does not permission to archive. So, this check on frontend is useless. Also, if a user gains access to send a request to archive , we will check the authenticity in the backend itself and send error code and error messsage.

Reply:
Done the changes.